src/stirling/source_connectors/socket_tracer/uprobe_manager.cc

```diff
@@ -282,53 +283,94 @@ StatusOr<std::vector<std::filesystem::path>> FindHostPathForPIDLibs(
                                 HostPathForPIDPathSearchType::kSearchTypeEndsWith);
 }
 
+// SSLLibMatcher allows customizing the search of shared object files
+// that need to be traced with the SSL_write and SSL_read uprobes.
+// In dynamically linked cases, it's likely that there are two
+// shared libraries (libssl and libcrypto). While statically linked
+// cases are contained within the same binary. SSLLibMatcher
```

Comment:
nit: "While statically linked" --> "In contrast, statically linked"

Reply:
Updated